static/js/load_map.js

```diff
@@ -75,4 +76,60 @@ function initialize() {
             markers.push(nextMarker);
         });
     });
+
+    const stores = document.getElementById('Stores');
+    function getLocation() {
+        if (navigator.geolocation) {
+            return navigator.geolocation.getCurrentPosition(showPosition);
+        } else {
+            x.innerHTML = "Geolocation is not supported by this browser.";
+        }
+    }
+
+    function showPosition(position) {
+        x.innerHTML = "Latitude: " + position.coords.latitude +
+        "<br>Longitude: " + position.coords.longitude;
+    }
+
+    function callback(results, status) {
+      if (status == google.maps.places.PlacesServiceStatus.OK){
+        markers.forEach(marker => {
+            marker.setMap(null);
+        });
+        markers = [];
```

Comment:
Yeah fair enough; at the very least I can move it inside the `initialize` function so it's block-scoped to the function as opposed to being global.